cit-api/pipeline/migrations/0139_usertracking.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.16 on 2021-03-29 20:53
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pipeline', '0138_auto_20210329_1530'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserTracking',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('report_url', models.CharField(max_length=2048)),
+                ('date_viewed', models.DateTimeField(auto_now=True)),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to='pipeline.User')),
```

Comment:
Setting on_delete to DO_NOTHING will probably prevent us from deleting users, as I think a IntegrityError will be thrown if there's a dangling foreign key reference from UserTracking to User. It will work for now, but might cause an issue later on.

Reply:
Fortunately we are soft deleting user such that we can track user opportunity relationships until the opportunity is force deleted.